methods.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+class DetailedProduct {
+//    public $id = 1;
+    public $title = 'Robot Apron';
+    public $price = 23;
+    public $image = 'https://images.unsplash.com/photo-1593376853899-fbb47a057fa0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2758&q=80';
+    public $imageTwo = 'https://images.unsplash.com/photo-1593376853899-fbb47a057fa0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2758&q=80';
+    public $imageThree = 'https://images.unsplash.com/photo-1593376853899-fbb47a057fa0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2758&q=80';
+    public $category = [
+        'id' => 1,
+        'name' => 'Destroyer of Worlds'
+    ];
+    public $character = [
+        'id' => 2,
+        'name' => 'Fred',
+        'image' => 'https://images.unsplash.com/photo-1593376853899-fbb47a057fa0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2758&q=80',
+        'description' => 'Lorem ipsum, or lipsum as it is sometimes known, is dummy text used in laying out print, graphic or web designs. The passage is attributed to an unknown typesetter in the 15th century who is thought to have scrambled parts of Ciceros De Finibus Bonorum et Malorum for use in a type specimen book.'
+    ];
+//    public $categoryID = 1;
+//    public $characterID = 2;
+    public $productDescription = 'This is a nice Apron';
+
+    public function displayDetailedProduct(): string {
```

Comment:
the agreed architecture was to have this in a loop on the product page itself. This is an additional file & and additional unit test to the outline drawn up. Is there a reason why this is on another page?